specs/BadgeNotifications/BadgeNotifications-spec.md

```diff
@@ -0,0 +1,222 @@
+# Badge Notifications in Windows App SDK
+
+This is the spec for proposal: [Issue #138](https://github.com/microsoft/WindowsAppSDK/issues/138)
+
+# Background
+
+A notification badge conveys summary or status information specific to your app. They can be numeric
+(1-99) or one of a set of system-provided glyphs. Examples of information best conveyed through a
+badge include network connection status in an online game, user status in a messaging app, number of
+unread mails in a mail app, and number of new posts in a social media app.
+
+Example of a numeric badge in Taskbar (One new message in Whatsapp):
+
+![Screenshot](NumericBadgeExample.png)
+
+Example of a system-provided glyph in taskbar (New message in Outlook):
+
+![Screenshot](GlyphBadgeExample.png)
+
+This Badge notification functionality is missing in WinAppSDK, which not only blocks apps like
```

Comment:
Would help to explain here why this API is different than the existing [BadgeNotification](https://docs.microsoft.com/uwp/api/Windows.UI.Notifications.BadgeNotification) API. (I'm guessing it's because the existing thing is more complex than is necessary anymore, but that should be documented here for the record.)

Reply:
Note is added in the spec